src/lib/layouts/Section.svelte

```diff
@@ -1,6 +1,6 @@
-<section id={$$props.id} class="py-10 child:mx-10 md:child:mx-20">
+<section id={$$props.id} class="py-10 child:mx-10 md:child:mx-20 {$$props.class}">
```

Comment:
`{$$props.class}` I don't think it's necessary

Reply:
You're right sorry